ibm_catalog.json

```diff
@@ -129,27 +135,15 @@
               }
             },
             {
-              "key": "provider_visibility",
-              "options": [
-                {
-                  "displayname": "private",
-                  "value": "private"
-                },
-                {
-                  "displayname": "public",
-                  "value": "public"
-                },
-                {
-                  "displayname": "public-and-private",
-                  "value": "public-and-private"
-                }
-              ]
```

Comment:
No need to delete this. Just add `hidden:true`

Reply:
sure. Done.

Reply:
@maheshwarishikha - What should be the correct value here as you mentioned [here](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/pull/83/files#r2097193891)  that the value should not be hidden i.e. should be `false` or maybe remove `hidden:true` itself ?